dissect/cstruct/parser.py

```diff
@@ -86,10 +86,11 @@ def _constant(self, tokens: TokenConsumer) -> None:
         except (ValueError, SyntaxError):
             pass
 
-        try:
-            value = Expression(self.cstruct, value).evaluate()
-        except Exception:
-            pass
+        if isinstance(value, str):
```

Comment:
Isn't a literal value a very basic form of an expression?

Reply:
Can you elaborate?

Reply:
Well I thought you could capture the simple case of a literal value or reference as being an expression as well, thus moving all logic to the Expression class. But to be honest, after looking at my own code / parser, I didn't do it either.

Reply:
In our case it's also a _massive_ performance hit if we would do that. The `Expression` class is easily the slowest part of reading structures.